.gitignore

```diff
@@ -1 +1,5 @@
 .idea
+node_modules/
```

Comment:
dist is also going to be covered through node_modules? If not, then add it here

Reply:
I will add dist as well. It won't be covered by `node_modules`.